stdlib/public/runtime/Reflection.mm

```diff
@@ -497,7 +486,18 @@ static Mirror reflect(HeapObject *owner,
   ::new (&result) MagicMirror(owner, mirrorValue, mirrorType);
   return result;
 }
+
+// -- Tuple destructuring.
   
+SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C"
+intptr_t swift_TupleMirror_count(HeapObject *owner,
+                                 const OpaqueValue *value,
+                                 const Metadata *type) {
+  auto Tuple = static_cast<const TupleTypeMetadata *>(type);
+  return Tuple->NumElements;
+}
+
```

Comment:
It feels more natural to put the tuple functions together, after the '// -- Tuple destructuring.' section. Previously swift_TupleMirror_count came first, then some other unrelated functions, then the rest of the TupleMirror stuff.

If you prefer I can drop this patch, it's not related to anything else in the series. Arguably moving code around destroys 'git blame' history, but I have some other changes to this file that are not yet committed, so I decided to fix this nit.